src/index.js

```diff
@@ -46,32 +47,23 @@ const displayTaskElement = (task) => {
   return taskItem;
 };
 
-document.getElementById('add-task-btn').addEventListener('click', () => {
-  const taskInput = document.getElementById('task-input');
-  const taskName = taskInput.value.trim();
-  if (taskName !== '') {
-    createTaskElement(taskName, tasksLocal);
-    document.location.reload();
-    taskInput.value = '';
-  }
-});
-
-function activateDeleteListener(delBtn, parent) {
-  delBtn.addEventListener('click', () => {
-    deleteTaskElement(tasksLocal, parent, delBtn);
-  });
-}
-
 function activateMoreListeners() {
   const moreBtn = document.querySelectorAll('.more-icon');
   moreBtn.forEach((mb) => {
     mb.addEventListener('click', (e) => {
       const clickedBtn = e.target;
       const parent = clickedBtn.parentNode;
       const delBtn = parent.getElementsByClassName('delete-icon')[0];
+      const taskList = document.getElementById('task-list');
       if (delBtn.classList.contains('hide')) {
         delBtn.classList.remove('hide');
-        activateDeleteListener(delBtn, parent);
+        delBtn.addEventListener('click', () => {
+          deleteTaskElement(tasksLocal, delBtn);
+          loadTasksFromLocalStorage();
+          taskList.innerHTML = '';
+          /* eslint-disable no-use-before-define */
```

Comment:
- [ ] It's not a good approach to disable linters, fixing linters help us to write a clean and strong codebase 🙃 🙃